R/count.R

```diff
@@ -624,9 +624,15 @@ process_count_denoms <- function(x) {
       abort("A value(s) were set with 'denom_ignore' but no missing count was set. Your percentages/totals may not have meaning.")
     }
 
-    # Logic to determine how to subset target for denominators
+    # Logic to determine how to subset target for denominators.
     if (is.null(denom_where)) {
-      denom_where <- where
+      # If a pop_data was passed change the denom_where to the pop_data_where
+      if (!isTRUE(try(identical(pop_data, target)))) {
+        denom_where <- quo(TRUE)
```

Comment:
Not in the case where there is a pop_data. In that case the `pop_where` is used and the `denom_where` is ignored. We have this message below to note this behavior.
"A `denom_where` has been set with a pop_data. The `denom_where` has been ignored. You should use `set_pop_where` instead of `set_denom_where`."